### ISSUE TITLE
Install problem python3.10

### ISSUE THREAD

**Issue by makeclean** [mit-crpg/OpenMOC#475]
Hi, not sure if there is a better forum for this. On python3.10 install seems to work fine, but running a problem
```
13:59 $ python3 point-source-2d.py 
Traceback (most recent call last):
  File "/home/adavis/opt/OpenMOC/sample-input/point-source/point-source-2d.py", line 1, in <module>
    import openmoc
  File "/home/adavis/.local/lib/python3.10/site-packages/openmoc/__init__.py", line 18, in <module>
    from openmoc.openmoc import *
ModuleNotFoundError: No module named 'openmoc.openmoc'

```

**Comment by GiudGiud** [comment]
Hello

Can you please paste the install log?
Also did you run the `setup.py install` script twice? It was finicky like that

Guillaume

**Comment by makeclean** [comment]
I didn't run it twice, are you saying I should?

**Comment by GiudGiud** [comment]
yeah you have to the very first time

**Comment by makeclean** [comment]
Different error this time
```
Traceback (most recent call last):
  File "/home/adavis/opt/OpenMOC/sample-input/point-source/point-source-2d.py", line 1, in <module>
    import openmoc
  File "/home/adavis/.local/lib/python3.10/site-packages/openmoc/__init__.py", line 19, in <module>
    from openmoc.subdivider import Subdivider
  File "/home/adavis/.local/lib/python3.10/site-packages/openmoc/subdivider.py", line 9, in <module>
    import openmoc.checkvalue as cv
  File "/home/adavis/.local/lib/python3.10/site-packages/openmoc/checkvalue.py", line 2, in <module>
    from collections import Iterable
ImportError: cannot import name 'Iterable' from 'collections' (/usr/lib64/python3.10/collections/__init__.py)
```

**Comment by GiudGiud** [comment]
seems some import changed. needs to be from collections.abc
I doubt there's too many occurences if you want to try fixing it
https://issueexplorer.com/issue/Kozea/pygal/524

**Comment by tjlaboss** [comment]
Resolved with #476.
